### PR TITLE
feat: add support for `.cjs` configs

### LINKED ISSUE #60: Add support for `.cjs` configs
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `simple-git-hooks@2.5.1` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/simple-git-hooks/simple-git-hooks.js b/node_modules/simple-git-hooks/simple-git-hooks.js
index 772b9a0..44cccbc 100644
--- a/node_modules/simple-git-hooks/simple-git-hooks.js
+++ b/node_modules/simple-git-hooks/simple-git-hooks.js
@@ -131,7 +131,9 @@ function _getConfig(projectRootPath) {
         throw TypeError("Check project root path! Expected a string, but got " + typeof projectRootPath)
     }
     const sources = [
+        () => _getConfigFromFile(projectRootPath, '.simple-git-hooks.cjs'),
         () => _getConfigFromFile(projectRootPath, '.simple-git-hooks.js'),
+        () => _getConfigFromFile(projectRootPath, 'simple-git-hooks.cjs'),
         () => _getConfigFromFile(projectRootPath, 'simple-git-hooks.js'),
         () => _getConfigFromFile(projectRootPath, '.simple-git-hooks.json'),
         () => _getConfigFromFile(projectRootPath, 'simple-git-hooks.json'),
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

### PR DESCRIPTION
**Pull request by JounQin**
close #60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ A tool that lets you easily manage git hooks
 
 > The package was recently renamed from `simple-pre-commit`. 
 
-> See **Releases** for the `simple-pre-commit` documentation and challengelog
+> See **Releases** for the `simple-pre-commit` documentation and changelog
```
Comment (toplenboren) [inline README.md:10]: 🤟

**Review by toplenboren — APPROVED** [review]
LGTM!

**Comment by JounQin** [comment]
@toplenboren Any time to release?

**Comment by toplenboren** [comment]
> @toplenboren Any time to release?

Released in `2.6.1` :-)

**Pull request merged**